src/components/MultiHopTrade/components/AssetChainDropdown.tsx

```diff
@@ -117,7 +100,7 @@ export const AssetChainDropdown: React.FC<ChainDropdownProps> = ({
           )}
           {renderChains}
         </MenuOptionGroup>
-      </MenuList>
+      </MenuList> */}
```

Comment:
commented out since we want to rework this in follow-up, but note above, this menu is hardcoded `disabled` so is never actually rendered.